tests/Integration/apis/kms/v1/cloudkms_test_mixins_v1.yaml

```diff
@@ -0,0 +1,75 @@
+type: google.api.Service
+config_version: 3
+name: cloudkms.googleapis.com
+title: Cloud Key Management Service (KMS) API
+
+apis:
+- name: google.cloud.kms.v1.KeyManagementService
```

Comment:
Should this have the google.iam.v1.IAMPolicy mixin?

Reply:
Fixed. Keeping Locations in and leaving IAM out, to ensure that (a) this list is actually parsed, and (b) Locations has both regular and paginated RPCs.